fl4health/clients/basic_client.py

```diff
@@ -874,6 +917,20 @@ def get_data_loaders(self, config: Config) -> Tuple[DataLoader, ...]:
         """
         raise NotImplementedError
 
+    def get_test_data_loader(self, config: Config) -> Optional[DataLoader]:
+        """
+        User defined method that returns a PyTorch Test DataLoader
```

Comment:
I would expand on this comment a bit to say that by default, this function returns None, assuming that there is no test dataset to be used. If the user would like to load and evaluate a dataset, they need only override this function in their client class.